### ISSUE TITLE
Cloudflare bot detection

### ISSUE THREAD

**Issue by DutchXYZ** [perfilrobo/hexbot#46]
Could you perhaps add a customizable wait time when crawling + uploading? I'm hitting the bot detection every 5 minutes :/ Running on 1GBPs internet (real life) so it's ultra fast...

**Comment by perfilrobo** [comment]
Yeah. The cloudflare (or something, I don't know) can detect repetitive pattern. There's nothing to do about that.
